pkg/spoke/controllers/constants.go

```diff
@@ -2,6 +2,10 @@ package controllers
 
 const (
 	// ManifestWorkFinalizer is the name of the finalizer added to manifestworks. It is used to ensure
-	// all maintained resources of a manifestwork are deleted before the manifestwork itself is deleted
+	// related appliedmanifestwork of a manifestwork are deleted before the manifestwork itself is deleted
 	ManifestWorkFinalizer = "cluster.open-cluster-management.io/manifest-work-cleanup"
+	// AppliedManifestWorkFinalizer is the name of the finalizer added to appliedmanifestwork. It is to
+	// ensure all resource relateds to appliedmanifestwork is deleted before appliedmanifestwork itself
+	// is deleted.
+	AppliedManifestWorkFinalizer = "open-cluster-management.io/applied-manifest-work-cleanup"
```

Comment:
scope this to the `work.open-cluster-management.io/applied-manifest-work-cleanup`

Reply:
fixed